src/Controls/src/Core/Platform/AlertManager/AlertManager.Windows.cs

```diff
@@ -37,32 +37,33 @@ internal void Unsubscribe(Window window)
 
 		internal sealed class AlertRequestHelper : IDisposable
 		{
-			static Task<bool>? CurrentAlert;
-			static Task<string?>? CurrentPrompt;
+			Task<bool>? CurrentAlert;
+			Task<string?>? CurrentPrompt;
```

Comment:
@PureWeen this was causing the issue. No need for statics anymore! We have multiple windows!